.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# @tbaut owns any files in the /.github/
+
+# directory at the root of the repository and any of its
+
+# subdirectories.
+
+/.github/ @tbaut
```

Comment:
Note that this shouldn't change much in terms of security. I did setup this repo to enforce my approval for any change in the .github dir, but there are probably other ways to exfiltrate our secrets than to just change these.